framework/UPGRADE.md

```diff
@@ -15,6 +15,12 @@ php composer.phar self-update
 php composer.phar global require "fxp/composer-asset-plugin:^1.2.0"
 ```
 
+Upgrade from Yii 2.0.9
+----------------------
+
+* RBAC: Interface `ManagerInterface` has been changed to added `getRolesByRole()` method. If you create your Manager via
```

Comment:
`getRolesByRole()` method was added to `ManagerInterface`. If you've implemented your own RBAC manager you need to implement new method.